src/util.rs

```diff
@@ -104,10 +104,10 @@ where
 {
     let source = match file_path {
         Some(msg) => msg.display().to_string(),
```

Comment:
Possible a if let else construct is more readable than a match to check an Option. Something like

```
if let Some(path) = file_path { path} else { "(from stdin)"}